control-plane/grpc/proto/v1/node/node.proto

```diff
@@ -32,7 +38,8 @@ message NodeSpec {
   common.StringMapValue labels = 3;
   // Cordon information.
   NodeCordon cordon = 4;
-
+  // Drain information.
+  NodeDrain drain = 5;
```

Comment:
Interesting that we've laid out the cordon and drain in separate "containers" at the grpc layer, but we've added them to the parent type directly on the openapi, I wonder if the openapi should do the same, wdyt?

Reply:
TBH I was following Paul's pattern, not knowing any better.
In the interest of simplicity I'd prefer to go the other way and have NodeCordon and NodeDrain as repeated strings in grpc, if we can do that.